### DISCUSSION TITLE
How are non-parent-child nodes associated and described in mindmap mode for plantuml?

### DISCUSSION THREAD

**Discussion by Amoscey** [plantuml/plantuml#1043] [Q&A]
How are non-parent-child nodes associated and described in mindmap mode for plantuml?
```
@startmindmap
* 1
** 2.1
*** 3
** 2.2
@endmindmap
```
For example, I want to link 1 to 3 and then add relationship 4 to the line between 1 and 3.
Like this
![image](https://user-images.githubusercontent.com/101571163/174353287-5623154e-86fa-49bc-ac4f-6bbe4da679cd.png)

If someone knows about the above features, I'll be very grateful to be told how to do it. If the program doesn't have this feature now, I hope it can be added later.

**Comment by The-Lum** [comment]
[Just for traceability], see similar request here:
- https://forum.plantuml.net/10965/non-hierarchical-cross-links-in-mind-maps
- https://forum.plantuml.net/16038/mindmap-lines-over-the-hierarchy

Regards.
